j2k/testData/fileOrElement/switch/caseWithDefaultblock.java

```diff
@@ -0,0 +1,21 @@
+//method
+void foo() {
```

Comment:
Please try to ensure that testdata files are valid Java code. This file does not compile ('status' is undefined, the types do not match, etc.)